crates/common/tedge_config/src/tedge_toml/tedge_config.rs

```diff
@@ -2113,7 +2117,8 @@ mod tests {
         let error = tedge_config
             .mapper_config::<C8yMapperSpecificConfig>(&profile_name(None))
             .await
-            .unwrap_err();
+            .err()
```

Comment:
I see that `MapperConfig` not being `Debug` prompted this change. Any reason why we can't add the `Debug` bound to `SpecialisedCloudConfig` so that `MapperConfig` can also be `Debug`?